lte/gateway/c/core/oai/tasks/sgw/spgw_state_manager.cpp

```diff
@@ -93,13 +96,13 @@ void SpgwStateManager::free_state() {
     return;
   }
 
-  if (hashtable_ts_destroy(state_ue_ht) != HASH_TABLE_OK) {
-    OAI_FPRINTF_ERR(
-        "An error occurred while destroying SGW s11_bearer_context_information "
-        "hashtable");
+  if (state_ue_map.map && state_ue_map.destroy_map() != PROTO_MAP_OK) {
+    OAI_FPRINTF_ERR("An error occurred while destroying SPGW's state ue map ");
```

Comment:
I think we need to keep "OAI_FPRINTF_ERR" because free_state() would be called when tasks exits. So can't rely on logging task as it might have already exited. 
Let me change at other places in this function.